spec/features/applications/show_spec.rb

```diff
@@ -267,6 +268,24 @@
           expect(page).to_not have_content('Add a pet to this application')
         end
       end
+
+      describe 'And I have not added any pets to the application' do
+        it 'Then I do not see a section to submit my application' do
+          visit "/applications/#{@nigel.id}"
+          expect(page).to_not have_content('Finalize Application')
+
```

Comment:
Excellent test, I checked out the branch and created a new application in Rails Server and everything worked as expected as well.